app/src/main/java/ru/yandex/practicum/contacts/presentation/sort/SortTypeUI.java

```diff
@@ -13,15 +16,18 @@ public SortTypeUI(@NonNull SortType sortType, boolean selected) {
         this.sortType = sortType;
         this.selected = selected;
     }
-
     public SortType getSortType() {
         return sortType;
     }
-
     public boolean isSelected() {
         return selected;
     }
 
+    //Шаг 2. Реализация интерфейса
+    @Override
+    public boolean theSameAs(SortTypeUI newItem) {
+        return this.sortType == newItem.getSortType();
```

Comment:
Здесь лучше тоже геттер использовать:
```suggestion
        return this.getSortType() == newItem.getSortType();
```